hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1179,4 +1179,12 @@ public void testDatabaseLocationWithSlashInWarehouseDir() {
 
     assertThat(database.getLocationUri()).isEqualTo("s3://bucket/database.db");
   }
+
+  @Override
+  @Test
+  public void listTablesForEmptyNamespace() {
```

Comment:
`TestHiveCatalog` is a subclass of  `CatalogTests`, but the error reporting is slightly different, so it seemed better to override `CatalogTests.listTablesForEmptyNamespace()` here rather than make it exception matching more complicated